boot/seal.go

```diff
@@ -445,21 +445,24 @@ func resealKeyToModeenvSecboot(rootdir string, model *asserts.Model, modeenv *Mo
 	if err != nil {
 		return err
 	}
-	if !needed {
-		logger.Debugf("fallback reseal not necessary")
-		return nil
-	}
+	if needed {
+		rpbcJSON, _ := json.Marshal(rpbc)
+		logger.Debugf("resealing (%d) to recovery boot chains: %s", nextCount, rpbcJSON)
```

Comment:
should this use nextFallbackCount ? do we have this problem elsewhere here?

Reply:
Seems to be the only place.